e2e/util_module_test/util_test1/test.yml

```diff
@@ -0,0 +1,5 @@
+query: >
+    CALL util_module.md5(["string",[1,2,3]]) YIELD result RETURN result; 
+
+output:
+  - result: 7c45115a3283a86ecc2ad9b1b678cce0
```

Comment:
Can you make use of online tests, to use try multiple times to hash the same input to test we get same output? Check for example: e2e/node2vec_online_test/test_online_streamwalk_updater_missing
test to check how it works, and maybe we can have input as two queries which are empty, and for test call two times following input: 
CALL util_module.md5(["string",[1,2,3]]) YIELD result RETURN result;
to test we get both times same output.